distro/pom.xml

```diff
@@ -136,30 +136,43 @@
           </execution>
 
           <execution>
-            <id>Unpack EIP OpenMRS SENAITE routes to a temporary location</id>
-            <phase>generate-resources</phase>
+            <id>Copy EIP OpenMRS SENAITE routes to a temporary location</id>
+            <phase>package</phase>
             <goals>
-              <goal>unpack-dependencies</goal>
+              <goal>copy</goal>
             </goals>
             <configuration>
-              <excludeTransitive>true</excludeTransitive>
+              <artifactItems>
+                <artifactItem>
+                    <groupId>com.ozonehis</groupId>
+                    <artifactId>${eipOpenMRSSenaiteArtifactId}</artifactId>
+                    <version>${eipOpenMRSSenaiteVersion}</version>
+                    <type>jar</type>
+                    <overWrite>true</overWrite>
+                    <outputDirectory>${project.build.directory}/${project.artifactId}-${project.version}/binaries/${eipOpenMRSSenaiteArtifactId}</outputDirectory>
+                </artifactItem>
+              </artifactItems>
               <useBaseVersion>true</useBaseVersion>
-              <outputDirectory>${project.build.directory}/${eipOpenMRSSenaiteArtifactId}</outputDirectory>
-              <includeArtifactIds>${eipOpenMRSSenaiteArtifactId}</includeArtifactIds>
             </configuration>
           </execution>
 
           <execution>
-            <id>Unpack EIP Odoo OpenMRS routes in a temporary location</id>
-            <phase>generate-resources</phase>
+            <id>Copy EIP Odoo OpenMRS routes in a temporary location</id>
```

Comment:
Since you're copying now directly the JAR in the final folder, let's rename to: "Copy EIP Odoo OpenMRS JAR"